mdn/discord
# Fix #100: Measure installs with Postgres

#### Reviewing

Definitely go commit-by-commit. It's not that bad, but lots of lines were changed when `config.js` was modified and lots of files were moved when tests were added.
#### Testing

Follow the updated instructions in [TESTING.md](https://github.com/openjck/discord/blob/issue-100-measure-installs/docs/TESTING.md).
#### How to spot-check
1. Push this code up to your personal Heroku
2. `heroku addons:create heroku-postgresql:hobby-dev`
3. Delete your webhook from your test repository
4. Re-add your webhook
5. Verify that a ping was recorded
   1. `heroku pg:psql`
   2. `SELECT * FROM "Pings";`
